src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -44,34 +72,36 @@ TopicDomainHeader.propTypes = {
   handleOnMemberPress: PropTypes.func,
   hideSeeMember: PropTypes.bool,
   isFollow: PropTypes.bool,
-  memberCount: PropTypes.number
+  memberCount: PropTypes.number,
+  isLoading: PropTypes.bool
 };
 
 const styles = StyleSheet.create({
-  domainText: {
+  domainText: (isHeaderHide) => ({
     fontSize: normalizeFontSize(16),
     fontFamily: fonts.inter[600],
     textAlign: 'left',
-    color: colors.black,
+    color: isHeaderHide ? colors.white : colors.black,
     backgroundColor: 'transparent'
-  },
+  }),
   member: {
     width: normalize(16),
     height: normalize(16),
     marginRight: normalize(5)
   },
-  domainMember: {
+  domainMember: (isHeaderHide) => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[400],
     textAlign: 'left',
-    color: colors.blackgrey
-  },
-  seeMemberText: {
+    color: isHeaderHide ? colors.white : colors.blackgrey
+  }),
+  seeMemberText: () => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[500],
     textAlign: 'left',
-    color: colors.blue1
-  }
+    color: colors.signed_primary,
+    marginTop: normalize(1)
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color assignment in the `domainText` and `domainMember` styles could be simplified by using a single source of truth for the color value. This would make the code more maintainable and less prone to errors.
```diff
- color: isHeaderHide ? colors.white : colors.black,
+ color: colors[isHeaderHide ? 'white' : 'black'],
```
Do the same for `domainMember` style.


<!-- This is an auto-generated comment by OSS CodeRabbit -->